rwengine/src/script/modules/ObjectModule.cpp

```diff
@@ -29,6 +29,30 @@
 
 glm::vec3 spawnMagic( 0.f, 0.f, 1.f );
 
+/**
+ * @brief resolveModelID Resolves script & global model IDs to model id
+ * @param args
+ * @param argument
+ * @return
+ *
+ * Converts negative IDs into the model ID for the object that matches the model
+ * name at that index in the SCM's second section.
+ */
+int resolveModelID(const ScriptArguments& args, int argument)
```

Comment:
nit: I think I would have overloaded this for script args and only supported int arguments natively. But this works